### PR TITLE
TF-649: Improve Camera Masking Screen Performance

### PR DESCRIPTION
**Pull request by JordanBerni-UL**
- [ ] Code Review
- [ ] Developer Testing

### CODE REVIEW

File: TF_Settings_Web/src/Components/Pages/Camera/Masking/MaskingScreen.tsx [hunk 1]
```diff
@@ -162,62 +143,99 @@ const MaskingScreen = () => {
...
+            </>
+        ),
+        [mainLens.current]
+    );
+
+    const createMaskingOption = (input: MaskingOptionProps) => {
+        return (
+            <MaskingOption
+                title={input.title}
+                description={input.description}
```
Comment (cartmouse) [inline TF_Settings_Web/src/Components/Pages/Camera/Masking/MaskingScreen.tsx]: Can we remove this in favour of returning the JSX straight from the memos?

File: TF_Settings_Web/src/Components/Pages/Camera/Masking/MaskingScreen.tsx [hunk 2]
```diff
@@ -162,62 +143,99 @@ const MaskingScreen = () => {
...
             const handOne = hands[0];
             const handTwo = hands[1];
             const convertedHandOne = handOne ? handToSvgData(handOne, 0) : undefined;
             const convertedHandTwo = handTwo ? handToSvgData(handTwo, 1) : undefined;
 
-            setHandData({ one: convertedHandOne, two: convertedHandTwo });
+            handData.current = { one: convertedHandOne, two: convertedHandTwo };
         }
 
         // Inore any messages for a short period to allow clearing of message handling
```
Comment (cartmouse) [inline TF_Settings_Web/src/Components/Pages/Camera/Masking/MaskingScreen.tsx]: Typo in comment!

**Review by cartmouse — CHANGES_REQUESTED** [review]

File: TF_Settings_Web/src/Components/Pages/Camera/Masking/MaskingScreen.tsx (hunk 2 above)
Comment (JordanBerni-UL) [reply TF_Settings_Web/src/Components/Pages/Camera/Masking/MaskingScreen.tsx]: Good spot! Fixed this

File: TF_Settings_Web/src/Components/Pages/Camera/Masking/MaskingScreen.tsx (hunk 1 above)
Comment (JordanBerni-UL) [reply TF_Settings_Web/src/Components/Pages/Camera/Masking/MaskingScreen.tsx]: Using straight JSX now (with memo)

**Review by cartmouse — APPROVED** [review]
Looks good!

**Pull request merged**